spec/spec_helper.rb

```diff
@@ -18,7 +18,3 @@ def input(plugin, &block)
   end
 
 end
-
-RSpec.configure do |c|
-  c.include IrcHelpers
```

Comment:
+1
we need to find a way to namespace the includes. this is not just a devutils/plugin problem, but 2 plugins doing a conflicting include will also cause problems.
thanks for the input, I'm going to create the prs for the others